contents/blog/experiments.md

```diff
@@ -0,0 +1,106 @@
+
+It was a no brainer then to take the next step and introduce Experimentation. Since you're already analysing your data in PostHog, and are feature flagging new features, why not give the capability to test significant changes as well?
+
+We recently launched Experiments, and there's a ton we learned about running successful experiments. That's what this post is about.
+
+## The Peeking Problem is poorly named
+
+The strawman pop-sci version of the peeking problem goes something like: "You shouldn't look at experiment results while the experiment is running because that can lead to you ending experiments early, when the data is skewed in favour of one variant, thanks to random chance."
+
+However, peeking isn't the problem. The problem is taking action too quickly after peeking.
+
+We built this into our product: Peeking is fun, almost addicting, when you can see your experiment results changing in real time. It gives a sense of excitement, seeing your hypotheses being proven right or wrong. More importantly, it keeps you coming back to the experiment, tracking its progress.
+
+To solve the Taking-action-without-enough-information problem, we made it clear in our UI when it was okay to end an experiment: When results become significant, or the pre-determined duration for peeking has passed.[^1] This changed the conversation from 'peeking early and ending experiment if results look good' to 'waiting for the green light to switch on', and led to an overall much better experience.
+
+[^1]: If you're looking for how we calculate this, see the user guide (TK: link)
+
+
+## The kind of experiment you run determines how much data you need
+
+PostHog allows running two kinds of experiments:
+
+1. Trend experiments, which work on an absolute count. For example, a target metric like: "Number of people who clicked this specific button" would be a trend experiment. These kind of experiments deal with absolute numbers.
+
+2. Conversion experiments, which work on a relative percentage. These are the traditional popular kind of experiments. "Conversion rate for the sign up -> purchase flow"
+
+Perhaps surprisingly, trend experiments need more data and time to reach significance, versus conversion experiments. The reasoning is a bit complicated, but reach out if you're interested, we'll write an appendix! The short version is: You need 2 dimensions for both, test and control variants to accurately judge experiment results. In conversion experiments, these are people who entered the funnel, and people who exited the funnel. But for trends, you only have one count, the number of people who did the event. The second dimension for trends is exposure: a proxy for how long the experiment has been running. Since this parameter is completely dependent on time, these experiments tend to take a bit longer to reach significance. 
+
+(TK: well, all 4 params are dependent on time in a way, soooo.... this is a bit confusing)
+
+## Choosing the right metric is hard
```

Comment:
the next 3 headings flow smooth AF into each other. Definitely want to keep these 3 together, but the 2 on top don't seem to fit as well. Hmm.